src/routes/people/+page.svelte

```diff
@@ -12,25 +16,89 @@
     import eyronTanael from '$lib/people/eyron-tanael.webp';
     import hansLorico from '$lib/people/hans-lorico.webp';
     import jellyRaborar from '$lib/people/jelly-raborar.webp';
+    import lino from '$lib/lino-hero.svg';
+
+    import { get } from '$lib/stores/color-scheme';
+
+    let pastExec = false;
+    const theme = get();
 </script>
 
-<article class="prose flex max-w-none flex-col gap-8">
-    <figure class="m-0 flex flex-col items-center">
-        <picture class="max-w-3xl">
-            <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
-            <source type="image/webp" srcset={execMedium} />
-            <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
-        </picture>
-        <figcaption>The 2023-2024 Executive Board</figcaption>
-    </figure>
-    <section class="not-prose flex flex-wrap justify-around gap-8">
-        <Exec src={calebBunye} alt="Caleb Bunye" pos="President" />
-        <Exec src={hansLorico} alt="Hans Lorico" pos="Vice President for Innovation" />
-        <Exec src={eyronTanael} alt="Eyron Tanael" pos="Vice President for Service" />
-        <Exec src={bastiOrtiz} alt="Basti Ortiz" pos="Director for Engineering" />
-        <Exec src={arianRey} alt="Arian Rey" pos="Director for Membership & Internals" />
-        <Exec src={jellyRaborar} alt="Jelly Raborar" pos="Director for Branding & Creatives" />
-        <Exec src={beaDiokno} alt="Bea Diokno" pos="Director for External Relations" />
+<section>
+    <section class="prose mb-8 flex max-w-none flex-col gap-8">
+        <figure class="m-0 flex flex-col items-center">
+            <!-- <picture class="max-w-3xl">
+                <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
+                <source type="image/webp" srcset={execMedium} />
+                <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
+            </picture> -->
+            <figcaption>The 2024-2025 Executive Board</figcaption>
+        </figure>
+        <section class="not-prose flex flex-wrap justify-around gap-8">
+            <Exec src={lino} alt="Andri Crisostomo" pos="President" />
+            <Exec src={lino} alt="Andri Crisostomo" pos="Vice President for Innovation" />
+            <Exec src={lino} alt="David Zamuco" pos="Vice President for Service" />
+            <Exec src={lino} alt="Victor Reyes" pos="Director for Engineering" />
+            <Exec src={lino} alt="Bianca Amurao" pos="Director for Membership & Internals" />
+            <Exec src={lino} alt="Clarisse Bucu" pos="Director for Branding & Creatives" />
```

Comment:
Just wanted to place my two cents here that we generally want to avoid directly using the ampersand because it may be syntactically ambiguous. We need to escape it by using `&amp;` instead. See [HTML Character Entities by w3schools](https://www.w3schools.com/html/html_entities.asp) for more details.
```suggestion
            <Exec src={lino} alt="Bianca Amurao" pos="Director for Membership &amp; Internals" />
            <Exec src={lino} alt="Clarisse Bucu" pos="Director for Branding &amp; Creatives" />
```

Reply:
Yeah let's adopt this.